engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -890,4 +880,14 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
 
     runTransaction(action)
   }
+
+  private def materializeWorkflowDescriptor(id: WorkflowId, workflowSourceFiles: WorkflowSourceFiles)(implicit ec: ExecutionContext): Future[WorkflowDescriptor] = {
+    implicit val actorSystem = CromwellServer.actorSystem
+    val materializedWfDescriptorFuture =
+      PromiseActor.askNoTimeout(MaterializeWorkflowDescriptorActor.MaterializeWorkflow(id, workflowSourceFiles), CromwellServer.workflowDescriptorMaterializeActor).mapTo[MaterializationResult]
```

Comment:
That global actorsystem was a major pita and was only put in as a temporary hack leading up to PBE (months ago in the halcyon days when everyone assumed this would be a quick thing).

While I'm not a huge fan of the akka style implicit EC params, it might be a good use case here for the actor system.